subprojects/build-scan-performance/src/performanceTest/groovy/org/gradle/performance/BuildScanPluginPerformanceTest.groovy

```diff
@@ -62,6 +63,7 @@ class BuildScanPluginPerformanceTest extends AbstractBuildScanPluginPerformanceT
             invocationCount INVOCATIONS
             displayName(WITH_PLUGIN_LABEL)
             invocation {
+                distribution(distribution)
```

Comment:
Do you want to add this via `defaultSpec`?

Reply:
I don't understand this question - can you clarify pls?

Reply:
You can add this logic here: https://github.com/gradle/gradle/blob/52a2667afb325a6d009381f66d98fc6fb8e4fbd9/subprojects/build-scan-performance/src/testFixtures/groovy/org/gradle/performance/AbstractBuildScanPluginPerformanceTest.groovy#L66-L73

So not every spec does need to specify it again.

Reply:
fixed